src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/includeClassifier.test.js

```diff
@@ -0,0 +1,157 @@
+import includeClassifier from './includeClassifier';
+
+const vjIncludeSupportingAmp =
+  '/include/newsspec/21841-green-diet/gahuza/app?responsive=true&newsapps=true&app-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-asdf/app_launcher_ws_640_7ania.png&app-clickable=true&amp-clickable=true&amp-image-height=360&amp-image-width=640&amp-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-asdf/app_launcher_ws_640_7ania.png';
+const vjIncludeNotSupportingAmp =
+  '/news/special/2016/newsspec_14813/content/iframe/gahuza/us-gop.inc?responsive=true&app-clickable=true&app-image=http://a.files.bbci.co.uk/worldservice/live/assets/images/2016/11/09/161109092836_us_election_2nddaymaps_winner_ws_62_v3.png';
+const idt1Include = '/indepthtoolkit/quizzes/123-456';
+const idt2Include = '/idt2/111-222-333-444-555';
+
+const canonicalPathname = 'https://www.bbc.com/service/foo';
+const ampPathname = 'https://www.bbc.com/service/foo.amp';
+
+describe('Include Classifier', () => {
```

Comment:
Maybe we can group some the tests together in this file using the `it` approach you've found below? So maybe:
- A test for all include types on AMP
- A test for all include types on canonical

Reply:
These tests look great to me, thanks for taking the time, a love the `it.each` feature!